src/main/java/me/juancarloscp52/entropy/mixin/BlockMixin.java

```diff
@@ -45,6 +49,25 @@
 @Mixin(Block.class)
 public class BlockMixin {
 
+    @Inject(method = "updateEntityMovementAfterFallOn", at = @At("HEAD"), cancellable = true)
+    private void bounce(BlockGetter level, Entity entity, CallbackInfo ci) {
+        if(Variables.bouncyBlocks && !entity.isSuppressingBounce()) {
+            ci.cancel();
```

Comment:
Shouldn't this only be canceled when the movement is adjusted? Otherwise vanilla code will always be skipped during the event, even when it didn't change movement.